src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -0,0 +1,61 @@
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import { useState } from 'react';
+import StepOne from '../../../src/components/RegisterForm/StepOne';
+import StepTwo from '../../../src/components/RegisterForm/StepTwo';
+
+function RegisterForm() {
+  const [data, setData] = useState({
+    email: '',
+    companyName: '',
+    companyLogin: '',
+    companyPassword: '',
+    companyPassword2: '',
+    login: '',
+    password: '',
+    password2: '',
+  });
+  const [currentStep, setCurrentStep] = useState(0);
+
+  // Fake Request
```

Comment:
I would remove this comment TBH. It doesn't bring any valued knowledge with itself